### DISCUSSION TITLE
有几处问题

### DISCUSSION THREAD

**Discussion by yumianhuli2** [SWHL/RapidVideOCR#54] [Q&A]
调用RapidVideoSubFinderOCR api，发现几处问题，求教如何解哈
1、个别1-2个字的字幕行无法识别
2、视频中第二行字幕都无法识别，但VSF是给完整了
举例：
![0_02_11_366__0_02_13_699_0000008641440021614401080](https://github.com/RapidAI/RapidOCR/assets/39210837/bffd7539-e346-4023-9b14-b27a021307ed)
00:02:11,366 --> 00:02:13,699
去翡翠公寓把方墨的东西全部给我搬过
# ‘来’这个字无法识别
3、对于识别错误的字，有换其他模型提高准确率的方法吗？
4、除了create_cleared_text_images设置为False之外，还有什么其他方法能缩小VSF和OCR的处理时间？

谢谢，大佬！

**Comment by SWHL** [comment]
丢失文字和识别错误的问题，是OCR模型问题。可以考虑使用sever版的检测和识别模型。详情参见：https://rapidai.github.io/RapidOCRDocs/docs/blog/01_optim_tutorial/

问题4：这个除非用GPU或者用其他在线OCR API

**Reply by yumianhuli2** [reply]
ok，3Q。VSF还可以用GPU吗？OCR我尝试过使用use_cuda=True，耗时反而更长咯

**Comment by SWHL** [comment]
如果想用GPU，就要安装rapidocr paddle版本。建议去看RapidOCR官方文档翻一下

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年03月28日 22:26 |
| 收件人 | SWHL/RapidVideOCR ***@***.***> |
| 抄送人 | SWHL ***@***.***>,
Comment ***@***.***> |
| 主题 | Re: [SWHL/RapidVideOCR] 有几处问题 (Discussion #54) |

ok，3Q。VSF还可以用GPU吗？OCR我尝试过使用use_cuda=True，耗时反而更长咯

**Reply by yumianhuli2** [reply]
感觉OCR识别时间不是瓶颈，即使是CPU也不慢，但VSF耗时比较长

**Reply by SWHL** [reply]
VSF这个算是极限了，C++编写的。似乎VSF也可以用GPU，可以调研一下
